lib/taurus/qt/qtgui/extra_guiqwt/tools.py

```diff
@@ -215,7 +215,7 @@ def __init__(self, manager, scrollFactor=0.2, toolbar_id=None):
 
     def register_plot(self, baseplot):
         ToggleTool.register_plot(self, baseplot)
-        self.connect(baseplot, SIG_ITEMS_CHANGED, self.items_changed)
+        baseplot.SIG_ITEMS_CHANGED.connect(self.items_changed)
```

Comment:
`guiqwt.signals.SIG_ITEMS_CHANGED` is actually `SIGNAL('items_changed(PyQt_PyObject)')`, so I guess it should be `baseplot.items_changed.connect`.